src/js/survis.js

```diff
@@ -168,7 +168,7 @@ var warnings = (function () {
     }
 
     function computeWholeFieldCapitalizationProtected(warningsList, field, value) {
-        if (value.indexOf('{') == 0 && value.lastIndexOf('}') == value.length - 1 && value.length > 10) {
+        if (value.indexOf('{') == 0 && value.lastIndexOf('}') == value.length - 1 && value.length > 10 && (value.split("{").length - 1 == 1) )  {
```

Comment:
Please dont use `==` in Javascript, always use `===`
https://stackoverflow.com/a/359509/1611511

Reply:
Thank you Hagen. I have updated the pull request.